### PR TITLE
Doc: filter duplicates and table of contents

### PR DESCRIPTION
**Pull request by jorgebay**

### CODE REVIEW

File: doc/TOC.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+The DataStax Node.js Driver for Apache Cassandra is a modern [feature-rich](#features) and highly tunable Node.js
```
Comment (tolbertam) [inline doc/TOC.md]: It doesn't look like there is a #features part of this page, did you mean to link to submodules?

Comment (tolbertam) [inline doc/TOC.md]: Ah i see, the tagline/link comes from the README.md page in our repo.

**Comment by tolbertam** [comment]
Looks great, only other comment I have is it'd be good to update README.md in the repo root to point to the TOC page root instead of Client.html (but only after this is deployed of course) for the API Docs link.

File: doc/TOC.md (hunk 1 above)
Comment (jorgebay) [inline doc/TOC.md]: Fixed, thanks!

**Pull request merged**